common/validation.go

```diff
@@ -487,6 +487,14 @@ func (c *CORSConfig) Validate() error {
 	if c.AllowedOrigins == nil || len(c.AllowedOrigins) == 0 {
 		return fmt.Errorf("*.cors.allowedOrigins is required, add at least one allowed origin")
 	}
+	for _, origin := range c.AllowedOrigins {
```

Comment:
@aramalipoor with this in context (https://github.com/erpc/erpc/blob/3526fec18f815681cdf4bc469116fb56a7db632f/erpc/http_server.go#L585), is it valid to have this strict validation here?

Reply:
I don't think this validation is correct, there could be cases when user wants to allow the RPC endpoint be used anywhere. Without "*" they can't do it as the browser will essentially require them to provide all the sites in the world.